addon/components/hyper-search.js

```diff
@@ -18,12 +18,14 @@ const {
  * Returns the key for the query in the cache. Only works in conjunction with
  * Ember.get.
  *
+ * Replaces perious (.) with dashes (-)
+ *
  * @public
  * @param {String} query
  * @return {String} nested key name
  */
 function keyForQuery(query) {
-  return `_cache.${query}`;
+  return query.replace('.', '-');
```

Comment:
I'm fine if this was updated to be `_cache.${query.replace('.', '-')}`, I would prefer this method to give you the actual key and not just a portion of it